src/app/campaign.model.ts

```diff
@@ -37,7 +37,7 @@ export class Campaign {
     public quotes: Array<{person: string, quote: string}>,
     public solution: string,
     public startDate: Date,
-    public status: 'Active' | 'Expired' | 'Preview' | 'Pending',
+    public status: 'Active' | 'Draft' | 'Expired' | 'Preview' | 'Pending',
```

Comment:
I think we can/should avoid adding a whole new logic branch separate from 'Preview' – will add more on the other PR